app.js

```diff
@@ -0,0 +1,22 @@
+import { myDataArray } from './myDataArray';
+
+// Need Id, Image, Name, Radio
+const idTags = document.querySelectorAll('id');
+const imageTags = document.querySelectorAll('img');
+const nameTags = document.querySelectorAll('name');
+const myRadioTags = document.querySelectorAll('input');
+// const data = new dataArray('myDataArray');
+
+let myWins = 0;
+let correctAnswer;
+
+myRadioTags.forEach((radioTag)) => {
```

Comment:
I'm guessing the problem is that you have an extra parenthesis. You have `myRadioTags.forEach((radioTag)) =>` but that extra closing parenthesis is one too many. Instead, try `myRadioTags.forEach((radioTag) =>`. Also make sure to always keep your `Problems` tab open in VSCode, because that'll tell you if you have any errors like this in the future.

Reply:
Thank you. Will do.